### PR TITLE
Return limit for assorted account methods

### PR DESCRIPTION
**Pull request by officialfrancismendoza**
**Issue** (#221): Limit parameter is not returned for account_lines, account_channels, account_objects, account_offers checks
**Fix**: Add limit parameter within response object for each respective file

### CODE REVIEW

File: test_scripts/clio_223.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+/*Script provided by Nathan Nichols*/
```
Comment (godexsoft) [inline test_scripts/clio_223.js]: Imho, not a great name for the script - `clio_223`. i suppose it is referring to the issue 223 but i would try call it something meaningful instead like `requestAccountChannels` for example. 

Another thing is - i'd probably not put these two scripts in the codebase at all. At the very least not until everyone agree we want to have those in the codebase and they should be in javascript. We already have one uberscript written in python called `test.py`. 

@cjcobb23 what do you think about tests scripts in the repo?

Comment (cjcobb23) [reply test_scripts/clio_223.js]: Yes let's not add new scripts to the repo. Pretty sure what you want to do is already handled in test.py. And I feel that it's questionable whether test.py should even be in the repo either. You can still use this script to test @officialfrancismendoza , but just don't add it to the repo.

**Comment by cjcobb23** [comment]
Please give the PR a descriptive title and link to the issue in the description.

**Review by godexsoft — DISMISSED** [review]
:+1:

**Review by cjcobb23 — DISMISSED** [review]
This needs to be squashed to one commit, with a good commit message, and then its good to go.

**Comment by officialfrancismendoza** [comment]
@godexsoft squashed to one commit. Can you review? If looks good, please approve & merge. Thank you! :)

**Review by godexsoft — DISMISSED** [review]
LGTM

**Review by godexsoft — DISMISSED** [review]

**Review by godexsoft — APPROVED** [review]

**Pull request merged**